Code/Framework/AzFramework/AzFramework/Archive/ZipDirCache.cpp

```diff
@@ -101,10 +101,11 @@ namespace AZ::IO::ZipDir
         FileEntry* operator -> () { return m_pFileEntry; }
         FileEntryTransactionAdd(Cache* pCache, AZStd::string_view szRelativePath)
             : m_pCache(pCache)
+            , m_szRelativePath("", AZ::IO::PosixPathSeparator)
```

Comment:
The Path classes have a constructor that accepts just a path separator, there is no situation where an empty string should ever be passed in to initialize a path.
```suggestion
            , m_szRelativePath(AZ::IO::PosixPathSeparator)
```

Reply:
Ah, I didn't know that.